extensions/wikia/FounderEmails/FounderEmails.class.php

```diff
@@ -70,12 +70,19 @@ public function getFoundersWithPreference( $prefPrefix ) {
 	 *
 	 * @param $user User
 	 * @param $event FounderEmailsEvent
+	 * @param $mailSubject
+	 * @param $mailBody
+	 * @param $mailBodyHTML
+	 * @param int $wikiId
+	 * @param string $category
+	 *
+	 * @return Status
 	 */
 	public function notifyFounder( $user, $event, $mailSubject, $mailBody, $mailBodyHTML, $wikiId = 0, $category = 'FounderEmails' ) {
 		global $wgPasswordSender, $wgNoReplyAddress;
 		$from = new MailAddress( $wgPasswordSender, 'Wikia' );
 		$replyTo = new MailAddress ( $wgNoReplyAddress );
-		if ( $event->enabled( $wikiId, $user ) ) {
+		if ( $event->enabled( $user, $wikiId ) ) {
```

Comment:
You'll see a bunch of updates for this.  I wanted to move the wiki ID to the end so it could be optional.  Its passed in a lot of placed, but I think a lot of times its just the current wiki.